app/models/dish.rb

```diff
@@ -18,16 +18,24 @@ class Dish < ActiveRecord::Base
   RATE_DIMENSION_DISH = 'dish'
   ratyrate_rateable RATE_DIMENSION_DISH
 
-  def rates
+  # public activity
+  include PublicActivity::Model
+  tracked owner: Proc.new{ |controller, model| controller.try(:current_user) }, recipient: Proc.new{ |controller, model| model.try(:user) } 
```

Comment:
Digging through the PublicActivity source, this is the method we have to ultimately satisfy: https://github.com/chaps-io/public_activity/blob/fc9c89ffa20662af83e57b5c326e9aff01cdf07b/lib/public_activity/common.rb#L5-L24

Will update shortly...